Automattic/jetpack
# Build: Remove the extensions directly before building them fresh

This helps to prevent abandoned files from sticking around between builds.
Fixes #12019

#### Changes proposed in this Pull Request:
* See above.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* If you're an Automattician, include a shortlink to the p2 discussion with Jetpack Product here.

#### Testing instructions:
* Before patch, build Jetpack and add a random file to `_inc/blocks. Run `yarn build` and see the new file still there.
* After patch, repeat and confirm the random file is no longer present.

#### Proposed changelog entry for your changes:
* Build Tools: Improve our build tools to help prevent things from falling through the cracks.
